### DISCUSSION TITLE
Wildcard in batches?

### DISCUSSION THREAD

**Discussion by thestorytree** [vladmandic/sdnext#3110] [General]
Is there a way a wildcard can run with the same wildcards, before shuffling them again and then doing a batch of the same number? So, it runs say 5 generations with the same wildcards selected, then moves onto 5 more with different wildcards selected, and so on in batches of 5.

**Comment by vladmandic** [comment]
that would really depend on how wildcards are implemented.

**Reply by thestorytree** [reply]
I have dynamic prompts extension - and say i have wildcards text file in the wildcards folder "backgrounds", with "forest", "beach", and "spaceship". Carrying on the example, can it choose the same background 5 times in a row, then choose something random for 5 times and so on? So the generations would go something like (spaceship, spaceship, spaceship, spaceship, spaceship, beach, beach, beach, beach, beach, forest, forest, forest, forest, forest, etc..)
